packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -20,6 +20,11 @@ import styles from './Cell.module.scss';
 
 const cx = classNames.bind(styles);
 
+const ColumnHighlightColor = {
```

Comment:
This enumeration should only be defined once.  Currently, there is more than one definition between Flowsheet and Table subcomponents.

Reply:
Removed here. 
Enum now defined in columnShape.js: https://github.com/cerner/terra-framework/pull/1985/commits/8a2aac51a800bf755283f26bbdb29313b3703c04